examples/translator/server/translator.cc

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "absl/strings/str_cat.h"
+#include "asylo/util/logging.h"
+#include "examples/translator/proto/translator.grpc.pb.h"
+#include "examples/translator/proto/translator.pb.h"
+#include "include/grpcpp/grpcpp.h"
+
+ABSL_FLAG(std::string, port, "7770", "Port on which this Server listens.");
```

Comment:
Perhaps mention "gRPC" in the description.

Reply:
(Aside: I'll aim to throw this test server away once there are proper tests of the gRPC client pseudo-Node in `abitest`)